src/app/index.html

```diff
@@ -9,8 +9,9 @@
 	<link rel="stylesheet" type="text/css" href="css/animation.css">
 	<link rel="stylesheet" type="text/css" href="../../node_modules/butter-component-settings/dist/styles.css">
 	<link rel="stylesheet" type="text/css" id="theme" href="themes/Official_-_Dark_theme.css">
+    <link rel="stylesheet" type="text/css" id="NewTheme" href="../../node_modules/butter-theme-dark/index.css">
```

Comment:
all indent is wrong: https://github.com/butterproject/butter-desktop/blob/xaiki/react/src/app/index.html#L12